include/lang/SC_LanguageClient.h

```diff
@@ -40,17 +41,17 @@ SCLANG_DLLEXPORT void destroyLanguageClient(class SC_LanguageClient*);
 class SCLANG_DLLEXPORT SC_LanguageClient {
 public:
     struct Options {
-        Options(): mMemSpace(2 * 1024 * 1024), mMemGrow(256 * 1024), mPort(57120), mRuntimeDir(0) {}
+        Options(): mMemSpace(2 * 1024 * 1024), mMemGrow(256 * 1024), mPort(57120) {}
```

Comment:
Just as a side note, this constructor can be omitted as well:
```
struct Options {
    int mMemSpace = 2 * 1024 * 1024; // memory space in bytes
    int mMemGrow = 256 * 1024; // memory growth in bytes     
    int mPort = 57120; // network port number
    std::string mRuntimeDir; // runtime directory
};
```
A little bit more verbose, but maybe clearer:
```
struct Options {
    static constexpr int defaultMemSpace = 2 * 1024 * 1024;
    static constexpr int defaultMemGrow = 256 * 1024;
    static constexpr int defaultPort = 57120;

    int mMemSpace = defaultMemSpace; // memory space in bytes
    int mMemGrow = defaultMemGrow; // memory growth in bytes     
    int mPort = defaultPort; // network port number
    std::string mRuntimeDir; // runtime directory
};
```